fe/fe-core/src/main/java/com/starrocks/http/HttpServerHandler.java

```diff
@@ -104,11 +105,11 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
                     long latency = System.currentTimeMillis() - startTime;
                     metrics.handlingRequestsNum.increase(-1L);
                     metrics.requestHandleLatencyMs.update(latency);
-                    LOG.info("receive http request. url: {}, thread id: {}, startTime: {}, latency: {} ms",
-                            req.getRequest().uri(), Thread.currentThread().getId(), startTime, latency);
-                    LOG.info("receive http request. uri: {}, thread id: {}, startTime: {}, latency: {} ms",
-                            WebUtils.sanitizeHttpReqUri(req.getRequest().uri()), Thread.currentThread().getId(),
-                            startTime, latency);
+                    if (latency > Config.http_slow_request_threshold_ms) {
```

Comment:
would be better if the request is slow,  log a warn else log a info log.

Otherwise we don't have any log to a success/fail http request.

Reply:
Fixed